src/program.ts

```diff
@@ -121,9 +121,12 @@ export class Program extends DiagnosticEmitter {
 
     this.types = new Map([
       ["i8", Type.i8],
+      ["byte", Type.i8],
```

Comment:
Actually, aliases like these were in the prototype compiler, along the lines of:

i8 = sbyte
u8 = byte
i16 = short
u16 = ushort
i32 = int
u32 = uint
f32 = float
f64 = double

That is, until this issue emerged: https://github.com/AssemblyScript/prototype/issues/104